assets/css/style.css

```diff
@@ -633,3 +633,118 @@ small {
     grid-template-columns: 40% 1fr;
   }
 }
+/* My CSS Code */
+
+/* #1 id selector for form background */
+#contact-form{
+  background: white;
+  padding: 1rem;
+}
+/* #2 container of the form to center it  */
+.container-form{
+  height: 100%;
+  margin: auto;
+}
+
+/* #3 the title in top of the form  */
+.title-form-top{
+  color: var(--text-gray);
+  font-size: 1.8rem;
+  text-align: justify;
+  text-justify: auto;
+  margin-bottom: 30px;
+}
+
+/* #4 editing the standard input to remove borders */
+input{
+  border: none!important;
+  background-color: var(--light-gray-blue);
+  color: var(--text-gray);
+}
+
+/* #5 The full section styling */
+.contact-form-section {
+  background-color: var(--light-gray-blue);
+  text-align: center;
+  font-family: Arial, sans-serif;
+  margin: 0 auto;
+  padding: 20px;
+  box-shadow: 0 0 15px rgba(0,0,0,0.1);
+  display: flex;
+}
+
+/* #6 First container relative */
+.container-image{
+  position: relative;
+}
+
+/* #7 image with display in block to ignore sibling */
+.form-image{
+  display: block;
+}
+
+/* #8 text set up to ignore sibling and sit in top of it with z-index */
+.title-form{
+  top:40%;
+  left:50%;
+  font-size: 2.8rem;
+  text-align: left;
+  font-weight: 100;
+  transform: translate(-50%,-50%);
+  position: absolute;
+  color: white;
+  z-index: 1;
+}
+/* #9 Grid of the form fields */
+.form-div{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+/* #10 separation of fields */
+.form-group {
+  margin-bottom: 20px;
+  display: grid;
+  margin: 1rem;
+}
+
+label {
```

Comment:
I noticed you have styling for `label`. However, I didn't see any elements with a class or id of `label` in the HTML. Is this CSS rule being applied as expected?